sdk/src/main/html/hcaptcha.html

```diff
@@ -0,0 +1,197 @@
+<!DOCTYPE html>
+<html>
+<head lang="en">
+    <title>hCaptcha Android</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no"/>
+    <style>
+        * {
+            padding: 0;
+            margin: 0;
+        }
+        body {
+            display: table;
+            width: 100%;
+            height: 100%;
+            text-align: center;
+        }
+        #hcaptcha-container {
```

Comment:
For some reason checkbox is not centered anymore.

![image](https://user-images.githubusercontent.com/11096226/201939060-d785c234-c466-4caf-8241-76c4b78f4f36.png)

Reply:
I pushed a fix https://github.com/hCaptcha/hcaptcha-android-sdk/pull/80/commits/b060cb44a43993602ac1d98777dd335e430a6b71